apps/user-office-frontend/src/components/proposal/ProposalTableInstrumentScientist.tsx

```diff
@@ -66,12 +66,34 @@ let columns: Column<ProposalViewData>[] = [
     removable: false,
     field: 'rowActionButtons',
   },
-  { title: 'Proposal ID', field: 'proposalId' },
+  {
+    title: 'Proposal ID',
+    field: 'proposalId',
+  },
   {
     title: 'Title',
     field: 'title',
     ...{ width: 'auto' },
   },
+  {
+    render: (principalInvestigator) => {

```

Comment:
This should be called `proposalView` or `rowData` maybe because it is a bit strange to see `principalInvestigator.principalInvestigator` in the bottom

Reply:
Yea good call. Thanks!